src/components/Tokens/TokenDetails/index.tsx

```diff
@@ -207,7 +219,7 @@ export default function TokenDetails({
       <TokenDetailsLayout>
         {detailedToken && !isPending ? (
           <LeftPanel>
-            <BreadcrumbNavLink to={`/tokens/${chain.toLowerCase()}`}>
+            <BreadcrumbNavLink to={(isInfoExplorePageEnabled ? '/explore' : '/tokens') + `/${chain.toLowerCase()}`}>
```

Comment:
nit. i think we prefer this way in the codebase, but i'm not sure if there's any practical benefit other than just style consistency

```suggestion
            <BreadcrumbNavLink to={`${isInfoExplorePageEnabled ? '/explore' : '/tokens'}/${chain.toLowerCase()}`}>
```